corehq/apps/domain/forms.py

```diff
@@ -596,6 +625,13 @@ def _save_account_confirmation_settings(self, domain):
             settings.confirmation_link_expiry_time = self.cleaned_data.get('confirmation_link_expiry')
             settings.save()
 
+    def _save_release_mode_setting(self, domain):
+        if self.cleaned_data.get("release_mode_visibility"):
+            setting_obj = AppReleaseModeSetting.get_settings(domain=domain.name)
+            setting_obj.is_visible = self.cleaned_data.get("release_mode_visibility")
+            setattr(setting_obj, "release_mode_visibility", self.cleaned_data.get("release_mode_visibility"))
```

Comment:
what is this doing?

Reply:
Not used now. It was used earlier to check whether to hide this setting, once enabled.